browser/android/speedreader/features.cc

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/android/speedreader/features.h"
+
+#include "base/feature_list.h"
+
+namespace speedreader {
+namespace features {
+
+BASE_FEATURE(kBraveAndroidSpeedReader,
+             "BraveAndroidSpeedReader",
+             base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
what's wrong with `brave/components/speedreader/features.h`?

this will be a mess if you'll decide to go this way.

Reply:
Agree, It's much better just to use existing feature.

Reply:
Updated. No modification to flags anymore.